app/views/admin/terms_of_service/drafts/show.html.haml

```diff
@@ -0,0 +1,19 @@
+- content_for :page_title do
+  = t('admin.terms_of_service.title')
+
+- content_for :heading do
+  %h2= t('admin.terms_of_service.title')
+  = render partial: 'admin/terms_of_service/links'
+
+= simple_form_for @terms_of_service, url: admin_terms_of_service_draft_path, method: :put do |form|
+  = render 'shared/error_messages', object: @terms_of_service
+
+  .fields-group
+    = form.input :text, wrapper: :with_block_label, input_html: { rows: 8 }
+
+  .fields-group
+    = form.input :changelog, wrapper: :with_block_label, input_html: { rows: 8 }
+
+  .actions
+    = form.button :button, t('admin.terms_of_service.save_draft'), type: :submit, name: :action_type, value: :save_draft, class: 'button button-secondary'
```

Comment:
Style only thing, applies to a bunch of spots will leave just once -- can we use the multi-line form for this? (ie, each option on own line) - makes future-diff changes easier to read, keeps us clean on linting line length limits, etc.

Reply:
My text editor fails to correctly syntax-color HAML with said line breaks, so I avoid that style.